src/test/java/org/folio/edge/core/EdgeVerticleTest.java

```diff
@@ -57,7 +58,7 @@
   private static final String apiKey = "Z1luMHVGdjNMZl9kaWt1X2Rpa3U=";
   private static final String badApiKey = "ZnMwMDAwMDAwMA==0000";
   private static final String unknownTenantApiKey = "Z1luMHVGdjNMZl9ib2d1c19ib2d1cw==";
-  private static final long requestTimeoutMs = 3000L;
+  private static final long requestTimeoutMs = 10000L;
```

Comment:
You really want to wait 10 seconds during the unit test?

Reply:
never mind, I see your note above now... I def. think this is a windows thing.

Reply:
For windows, yeah. I might be able to fine tune it to 7 or 8 seconds, but I'd run into frequent errors due to DNS lookup delays at 5 seconds and none at 10 seconds.